### PR TITLE
Fix/add gatecheck bundle summary

### PR DESCRIPTION
**Pull request by RevacommCharlie**
Fix to add a gatecheck --bundle-directory flag for the image-publish stage.  Previously, it was using the same --artifact-directory as the image-scan stage, and preventing this from being set correctly for the image-scan stage.

### CODE REVIEW

File: pkg/pipelines/image-publish.go [hunk 1]
```diff
@@ -112,12 +112,12 @@ func NewimagePublish(stdout io.Writer, stderr io.Writer) *imagePublish {
 			AntivirusFilename:					 "clamav-report.txt",
 			GatecheckBundleFilename:     "gatecheck-bundle.tar.gz",
 			GatecheckConfigFilename:     "gatecheck.yaml",
-			GrypeFilename:               "grype-scan.json",
```
Comment (BacchusJackson) [inline pkg/pipelines/image-publish.go:115]: keep as grype-image-scan, do the same to the sbom as well

File: pkg/pipelines/config.go [hunk 2]
```diff
@@ -61,6 +62,7 @@ func NewDefaultConfig() *Config {
 		},
 		Artifacts: ArtifactConfig{
 			Directory:                   ".artifacts",
+			BundleDirectory:             ".artifacts",
```
Comment (BacchusJackson) [inline pkg/pipelines/config.go]: Don't default to a .artifacts, do artifacts for both of these. In a CI environment, .artifacts is fine but on a local machine, I don't like the idea of creating a hidden directory

**Review by BacchusJackson — CHANGES_REQUESTED** [review]

**Review by BacchusJackson — APPROVED** [review]

File: pkg/pipelines/image-publish.go (hunk 1 above)
Comment (RevacommCharlie) [reply pkg/pipelines/image-publish.go:115]: This is updated now

File: pkg/pipelines/config.go (hunk 2 above)
Comment (RevacommCharlie) [reply pkg/pipelines/config.go]: This is changed to "artifacts" now

**Pull request merged**